third-party/eazy/v1/webhook.yaml

```
@@ -1,12 +1,12 @@
 asyncapi: '2.1.0'
 info:
   title: Web Hook Specifications
-  description: Web Hook Specifications
+  description: Webhook specifications
   version: 1.0.0
 servers:
   production:
     url: 78.110.226.11
-    description: Server address of tyntec, the events origin from
+    description: The server address of tyntec, the events origin from
```

```suggestion
    description: tyntec's server address, the event origin
```

---

@ridlees, I'm fine with the first change (`tyntec's`) but not sure about the second. Does it make any sense (`the event origin`)? The original meaning was (IIUIC) the events originate (`ate` is missing) from this server. Is the meaning after your change the same?

---

```suggestion
    description: tyntec's server address - from where events originate
```
Agree that the meaning gets blurry. I like the originate. 

```suggestion
    description:  tyntec's server IP address, used as source for Messaging events. 
```
Or we can use source here.

```suggestion
    description: tyntec's production server address 
```
But isn't in some way the "events originate" redundant? We can live without it :D So, in the end, we can use go with address or IP address. Will leave that on you.

---

@ridlees, I think that it's good to remind those not-so-bright users that this is the address that they can put in their firewall rules or whatever. And since I guess that the original author wanted to use "originate", I would stay with your first suggestion (plus `the` events).
```suggestion
    description: tyntec's server address - from where the events originate
```
Does that work for you?

---

>  description: tyntec's server address - from where events originate
Agree that the meaning gets blurry. I like the originate.

Agree, that is why it was one of the options.